pkgs/development/python-modules/asteroid-filterbanks/default.nix

```diff
@@ -17,23 +18,26 @@
   scipy,
 }:
 
-buildPythonPackage rec {
+buildPythonPackage {
   pname = "asteroid-filterbanks";
-  version = "0.4.0";
+  version = "0.4.0-unstable-2024-12-02";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "asteroid-team";
     repo = "asteroid-filterbanks";
-    tag = "v${version}";
-    hash = "sha256-Z5M2Xgj83lzqov9kCw/rkjJ5KXbjuP+FHYCjhi5nYFE=";
+    # TODO: 0.4.0 is not compatible with recent torch and numpy
+    rev = "74a13c69ba9858d57bd4936f18f3c3bd4b2d580c";
+    hash = "sha256-iTwqq5qyTDK9f0MG2XUvYUGSOPJ2I/X/Pxrf+n7o8Mk=";
   };
 
-  # np.float is deprecated
-  postPatch = ''
-    substituteInPlace asteroid_filterbanks/multiphase_gammatone_fb.py \
-      --replace-fail "np.float(" "float("
-  '';
+  patches = [
```

Comment:
can you vendor the patch?

this link can break

Reply:
We reference over 2000 patches from GitHub in this repo. Why would this one suddenly break?